e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -204,40 +200,28 @@ validate_password() {
 echo "Creating Service Principal (SP) for access to ADLA Gen2 used in Databricks mounting"
```

Comment:
ADLA should be ADLS

Reply:
ADLA was not me. It was there already when I arrived... but I fixed